addon/components/universal-export/template.hbs

```diff
@@ -25,3 +25,33 @@
       performExport=(action "performExport")}}
   {{/if}}
 {{/modal-view}}
+
+
+{{#modal-view hidden=hideInfluencerNetwork closeAction=closeInfluencerNetworkModal centered=true
+  borderlessHeader=true title="Influencer Network" container="body"}}
+  <div class="modal-body text-center">
+    <p class="text-size-7">
+      <b>Influencer has been added to your network</b>
+    </p>
+    <p class="text-size-5 text-color-default-lighter padding-left-md padding-right-md">
+      By adding this influencer to your list, it was automatically added to your Influencers Network. You can access and edit your network in your IRM.
+    </p>
+
+    <div>
+      <a {{action "closeInfluencerNetworkModal"}} target="_blank" class="upf-btn upf-btn--default margin-top-x-sm margin-bottom-x-sm margin-right-xx-sm">
+        Skip
+      </a>
+      <a href="http://help.upfluence.co" target="_blank" class="upf-btn upf-btn--primary margin-top-x-sm margin-bottom-x-sm">
```

Comment:
It's seems that the `https` not working, should we fix this ?

Reply:
Yeah I noticed, I don't think we ever redirect to https, but we should probably fix it

Reply:
So we don't forget https://github.com/upfluence/backlog/issues/168